content/500-platform/10-about.mdx

```diff
@@ -9,8 +9,8 @@ displayed_sidebar: platformSidebar
 
 The [Console](http://console.prisma.io/login) enables you to manage and configure your projects that use Prisma Data Platform (PDP) products, and helps you integrate them into your application:
 
-- [Accelerate](/accelerate/what-is-accelerate): Global database cache with scalable connection pooling
-- [Pulse](/pulse/what-is-pulse): A managed Change Data Capture (CDC) service that enables real-time database subscriptions
+- [Accelerate](/accelerate): Global database cache with scalable connection pooling
+- [Pulse](/pulse): A managed Change Data Capture (CDC) service that enables real-time database subscriptions
```

Comment:
Can we rephrase this to not be the "managed CDC" positioning, but something more easily understandable?

Reply:
Updated to: 

- [Accelerate](/accelerate): Speeds up your queries with a global database cache with scalable connection pooling.
- [Pulse](/pulse): Enables you to build event-driven and real-time applications by letting you stream database change events in a type-safe way.

Does that work better?